src/modules/chat/index.js

```diff
@@ -19,7 +19,7 @@ const badgeTemplate = (url, description) => `
         <img alt="Moderator" class="chat-badge bttv-chat-badge" src="${url}" alt="" srcset="" data-a-target="chat-badge">
         <div class="tw-tooltip tw-tooltip--up tw-tooltip--align-left" data-a-target="tw-tooltip-label" style="margin-bottom: 0.9rem;">${description}</div>
     </div>
-`;
+`.replace(/\n^(?!<)\s+(?=<)/gm, '');
```

Comment:
?

Reply:
Whoops, this was in the wrong place in general. However there does exist a bug with badges where extra space will create whitespace in a span. This will be posted in another PR.